Kernel/KerrGeoOrbit.m

```diff
@@ -56,10 +56,10 @@
 
 KerrGeoOrbitSchwarzDarwin[p_, e_] := Module[{t, r, \[Theta], \[Phi], assoc, consts, En,L,Q},
 
-t[\[Chi]_] := tSchwarzDarwin[p,e,\[Chi]];
-r[\[Chi]_] := rSchwarzDarwin[p,e,\[Chi]];
-\[Theta][\[Chi]_] := \[Theta]SchwarzDarwin[p,e,\[Chi]];
-\[Phi][\[Chi]_] := \[Phi]SchwarzDarwin[p,e,\[Chi]];
+t =Function[{Global`\[Chi]}, Evaluate[ tSchwarzDarwin[p,e,Global`\[Chi]] ], Listable];
+r =Function[{Global`\[Chi]}, Evaluate[ rSchwarzDarwin[p,e,Global`\[Chi]] ], Listable];
+\[Theta] =Function[{Global`\[Chi]}, Evaluate[ \[Theta]SchwarzDarwin[p,e,Global`\[Chi]] ], Listable];
```

Comment:
What's the reason for putting \[Chi] in the global context here?

Reply:
[Chi] is a named dummy variable of the function. If you don't use a symbol in the Global` context, the output will be much harder to read, and because it is just a dummy there are no side effects to the global context.

Reply:
Sounds reasonable. Is it expected that these pure functions would be publicly visible? Would it be better to have them as nicely formatted objects?

Reply:
Probably. Have a nicely formatted form of the output, would be nice. (But not essential.)